Examples/Example-macOS/Source/AppAuthExampleViewController.m

```diff
@@ -281,23 +282,23 @@ - (IBAction)authWithAutoCodeExchangeHTTP:(nullable id)sender {
     // performs authentication request
     __weak __typeof(self) weakSelf = self;
     _redirectHTTPHandler.currentAuthorizationFlow =
-        [OIDAuthState authStateByPresentingAuthorizationRequest:request
-                            callback:^(OIDAuthState *_Nullable authState,
-                                       NSError *_Nullable error) {
-      // Brings this app to the foreground.
-      [[NSRunningApplication currentApplication]
-          activateWithOptions:(NSApplicationActivateAllWindows |
-                               NSApplicationActivateIgnoringOtherApps)];
-
-      // Processes the authorization response.
-      if (authState) {
-        [weakSelf logMessage:@"Got authorization tokens. Access token: %@",
-                         authState.lastTokenResponse.accessToken];
-      } else {
-        [weakSelf logMessage:@"Authorization error: %@", error.localizedDescription];
-      }
-      [weakSelf setAuthState:authState];
-    }];
+      [OIDAuthState authStateByPresentingAuthorizationRequest:request
+                                             presentingWindow:self.view.window
+                                                     callback:^(OIDAuthState *_Nullable authState, NSError *_Nullable error) {
+        // Brings this app to the foreground.
```

Comment:
Check indentation here.  Should be two spaces.

Reply:
Done.